docs/ja/getting-started/example-datasets/tpch.md

```diff
@@ -678,8 +678,6 @@ ORDER BY
 ```
 
 ::::note
-2024年10月時点では、相関サブクエリのためクエリはそのままでは動作しません。対応する問題: https://github.com/ClickHouse/ClickHouse/issues/6697
-
```

Comment:
@matsumotomiki Hi! Could you please kindly check if I modified the Japanese docs correctly?